Dockerfile

```diff
@@ -62,7 +62,5 @@ RUN apt-get update && \
     docker-compose-plugin
 
 
-COPY . /experiment
-WORKDIR /experiment
-RUN /venv/bin/pip install --disable-pip-version-check -r requirements.txt
-ENTRYPOINT ["./report/docker_run.sh"]
+COPY ./report/docker_run.sh /docker_run.sh
```

Comment:
Since we purely rely on pulling the repo from GitHub, we won't have `./report/docker_run.sh` before cloning right? : )

Reply:
We need to put this script into the docker image so that it can run in docker before we can clone the repo when running.
This way every time we run the image it will have the latest repo (just not running the latest docker_run.sh).

Reply:
How about `clone` in the Dockerfile (so that have access to the script) and `pull` in `docker_run.sh`?

Reply:
Alternatively, we can have:
```
CMD git clone <this repo> && \
    cd <this repo> && \
    docker_run.sh
```

Reply:
Ah yes we can do this

Reply:
Oh speaking of this, I have an additional feature request:
Print the hash value of the HEAD commit before experiment starts. For example:
```
# At the beginning of docker_run.sh:
echo "Using OSS-Fuzz-Gen version: $(git rev-parse HEAD)"
```
This helps us to pinpoint the range of commits that are responsible for improvement/regression.

Reply:
Let's hold this for a bit because @erfanio is working on a more automated solution at #49

Reply:
Sound good